packages/material-renderers/src/complex/MaterialTableControl.tsx

```diff
@@ -354,21 +356,23 @@ const NonEmptyRowComponent = ({
                 </Grid>
               </Fragment>
             ) : null}
-            <Grid item>
-              <Tooltip
-                id='tooltip-remove'
-                title={translations.removeTooltip}
-                placement='bottom'
-              >
-                <IconButton
-                  aria-label={translations.removeAriaLabel}
-                  onClick={() => openDeleteDialog(childPath, rowIndex)}
-                  size='large'
+            {!disableRemove ? (
```

Comment:
The same appliedUiSchema options handling can be added to the root level of the table control. I think in general it would be good to add the props to the table control too, so it can be configured by reusing.